src/backend/app/Services/MiniGridDashboardCacheDataService.php

```diff
@@ -152,8 +152,8 @@ public function setData($dateRange = []) {
             foreach ($period as $d) {
                 $day = $d->format('o-W');
                 foreach ($ticketCategories as $tC) {
-                    $result[$day][$tC->label_name]['opened'] = 0;
-                    $result[$day][$tC->label_name]['closed'] = 0;
+                    $result[$day][$tC['label_name']]['opened'] = 0;
+                    $result[$day][$tC['label_name']]['closed'] = 0;
```

Comment:
I don't think this works. `$tC->label_name` is the correct way to access the `label_name` property. However, Larastan is failing to understand that `label_name` is indeed a property of `ticketCategories`. 

What about 

```
/**
 * @property string $label_name
 */
class TicketCategory extends BaseModel {
    use HasFactory;

    protected $table = 'ticket_categories';
}
```